### ISSUE TITLE
Automatically run build after configuration

### ISSUE THREAD

**Issue by dappelt** [interledgerjs/ilp-kit#144]
Some values from `env.list` are hard code into some source files during the build. Therefore, I propose that `npm run build` is executed automatically after `npm run configure` (which modifies `env.list`).

It was not apparent to me that some config values are being hard coded during the build, so I just spend some time figuring out why some configuration changes are seemingly without affect (thanks to @sharafian for helping with debugging).

**Comment by vhpoet** [comment]
@dappelt can you point me to hardcodes?

**Comment by dappelt** [comment]
I rerun `npm run configure` and changed the values for host name and ledger name (2nd & 3rd question). Then I `npm start`ed the ilp-kit and some of the HTML files were still referencing to the old host- and ledger name. Executing `npm run build` fixed the problem for me, so I assumed some values are hard-coded in HTML files. 

I tried to reproduce the behavior, but the HTML was updated as expected without rebuilding. Having that said, the problem might have been related to the browser caching some files and not with rebuilding.
